react/modules/tasks/functional-routing.md

```diff
@@ -15,27 +15,91 @@ Focus on routing and functional components. Remove any button or functionality w
 - Create custom hooks for shared logic
 - Add pagination with URL synchronization
 
-## Application Requirements
-
-1. **Pagination**:
-   - Add pagination for your item list (search results)
-   - Display current page in URL using query parameters (e.g., `?page=2`)
-   - Pagination should appear after receiving the list of items
-
-2. **Master-Detail View**:
-   - Main page displays search results
-   - On item click, split page into 2 sections:
-     - **Left**: Continue displaying search results
-     - **Right**: Display item details using React Router `<Outlet>`
-   - Show loading indicator while fetching details
-   - Add close control for details section
-   - The details part should be optional — by default, no item is selected when the page first loads
-   - Reflect page number and selected item in the URL (e.g., `/?page=2&details=1` or `/:page/:detailsId?`). Urls like `/1` or `/1/567` are valid.
-
-3. **About Page**:
-   - Create an About page with application author information
-   - Include a link to the RS School React course
-   - Add navigation to reach the About page from the main application
+## Functional Requirements (max **100 points**)
+
+### Feature 1: Pagination (**30 points**)
+
+**As a** user
+**I want** to navigate through pages of search results
+**So that** I can view more items and keep my place in the app
```

Comment:
'and keep my place in the app' => 'keep track of the viewed items'?
Not sure, but the initial verbiage sounds confusing :(